lib/maily/email.rb

```diff
@@ -105,5 +109,24 @@ def update_template(new_content, part = nil)
         f.write(new_content)
       end
     end
+
+    def versions
+      regexp = Regexp.new("^#{self.name}:")
+      self.mailer.emails.select do |email_key, _email|
+        email_key.match(regexp)
```

Comment:
since we don't care about matches, `match?` is way faster 👌🏼

Reply:
I think `match?` method is not supported for ruby versions previous than 2.4.6

https://apidock.com/ruby/String/match%3F

Reply:
`master` branch, which points to v2, will officially support 2.5+: https://github.com/markets/maily/blob/master/.travis.yml#L3 so we can do it! 💪🏼